packages/database/src/core/PersistentConnection.ts

```diff
@@ -321,15 +322,15 @@ export class PersistentConnection extends ServerActions {
     if (this.connected_ && this.authToken_) {
       const token = this.authToken_;
       const authMethod = isValidFormat(token) ? 'auth' : 'gauth';
-      const requestData: { [k: string]: any } = { cred: token };
+      const requestData: { [k: string]: unknown } = { cred: token };
       if (this.authOverride_ === null) {
         requestData['noauth'] = true;
       } else if (typeof this.authOverride_ === 'object') {
         requestData['authvar'] = this.authOverride_;
       }
-      this.sendRequest(authMethod, requestData, (res: { [k: string]: any }) => {
-        const status: string = res[/*status*/ 's'];
-        const data: string = res[/*data*/ 'd'] || 'error';
+      this.sendRequest(authMethod, requestData, (res: { [k: string]: unknown }) => {
+        const status: string = res[/*status*/ 's'] as string;
+        const data: string = res[/*data*/ 'd'] as string || 'error';
```

Comment:
Nit: You can probably drop the explicit type in the declaration.

Reply:
Done